addons/statustimers/main_ui.lua

```diff
@@ -435,13 +472,15 @@ module.render_main_ui = function(s, status_clicked, settings_clicked)
         ui.im_window = true;
         local item_width, _, text_dim = get_base_sizes():unpack();
         local player_status = party.get_player_status();
+        local is_targeting_buff_menu = get_menu_name() == 'menu    buff    '

```

Comment:
`'menu    buff    '` would be better stored in a module global (makes it less of a "magic number")